CommonMark/CommonMarkAdditionalFeatures.cs

```diff
@@ -21,6 +21,11 @@ public enum CommonMarkAdditionalFeatures
         StrikethroughTilde = 1,
 
         /// <summary>
+        /// The parser will recognize emphasis in indented code.
```

Comment:
Perhaps you could expand this comment a little? For example, give an example on when someone might be using this as well as describe exactly what will be supported in the indented code.